test/strictMode/output/umd/emptyImport.js

```diff
@@ -1,7 +1,7 @@
 (function (global, factory) {
-	typeof exports === 'object' && typeof module !== 'undefined' ? factory(require('foo'), require('polyfills')) :
-	typeof define === 'function' && define.amd ? define(['foo', 'polyfills'], factory) :
-	factory(global.bar)
-}(this, function (bar) { 'use strict';
+	typeof exports === 'object' && typeof module !== 'undefined' ? factory(require('polyfills'), require('foo')) :
+	typeof define === 'function' && define.amd ? define(['polyfills', 'foo'], factory) :
+	factory(global.__dep0__, global.bar)
```

Comment:
I believe we should pass a literal `undefined` when the import is bare to make it clearer to those reading the output, and to play it safe against accessing random properties on the global object.